src/Propel/Generator/Model/Table.php

```diff
@@ -786,7 +786,7 @@ public function setupReferrers($throwErrors = false)
 
             // check for incomplete foreign key references when foreign table
             // has a composite primary key
-            if ($foreignTable->hasCompositePrimaryKey()) {
+            if (!is_null($foreignTable) && $foreignTable->hasCompositePrimaryKey()) {
```

Comment:
👍 as it fixes [this specific case](https://github.com/propelorm/Propel2/pull/1259#discussion_r152598253)